dhis-2/dhis-api/src/main/java/org/hisp/dhis/datastore/DatastoreEntry.java

```diff
@@ -45,7 +45,7 @@
 @ToString( onlyExplicitlyIncluded = true )
 @EqualsAndHashCode( onlyExplicitlyIncluded = true, callSuper = true )
 @NoArgsConstructor
-public class KeyJsonValue extends BaseIdentifiableObject
+public class DatastoreEntry extends BaseIdentifiableObject
```

Comment:
Should we call this `DataStoreEntry` with capital S? The internet seems to think that splitting data store on two words is most common.

Reply:
Have no strong opinion. My thought and argument making it one word is that it has less potential confusing this with other things that have `Data` as a prefix. `Datastore` as a prefix is less ambiguous or more specific.